### ISSUE TITLE
Wrong type of appointment

### ISSUE THREAD

**Issue by Mael-Le-Garrec** [simonecid/Vaccinbot#5]
The script should check the total of appointments for `chronodose` instead of just taking the next available appointment.
Here for example there are no "chronodose" available but if the center has appointments in the 24h it will be wrongly taken into account.

Example:

```
{'appointment_by_phone_only': False,
 'appointment_count': 574,
 'appointment_schedules': [{'from': '2021-05-11T00:00:00+02:00',
                            'name': '1_days',
                            'to': '2021-05-11T23:59:59+02:00',
                            'total': 0},
                           {'from': '2021-05-11T00:00:00+02:00',
                            'name': '2_days',
                            'to': '2021-05-12T23:59:59+02:00',
                            'total': 0},
                           {'from': '2021-05-11T00:00:00+02:00',
                            'name': 'chronodose',
                            'to': '2021-05-12T23:59:59+02:00',
                            'total': 0},
                           {'from': '2021-05-11T00:00:00+02:00',
                            'name': '7_days',
                            'to': '2021-05-17T23:59:59+02:00',
                            'total': 0},
                           {'from': '2021-05-11T00:00:00+02:00',
                            'name': '28_days',
                            'to': '2021-06-07T23:59:59+02:00',
                            'total': 146},
                           {'from': '2021-05-11T00:00:00+02:00',
                            'name': '49_days',
                            'to': '2021-06-28T23:59:59+02:00',
                            'total': 574}],
 'departement': '38',
 'erreur': None,
 'gid': 'd253788',
 'internal_id': 'doctolib253788',
 'last_scan_with_availabilities': '2021-05-11T16:13:31.973605+02:00',
 'location': {'city': 'La Tour-du-Pin',
              'latitude': 45.5614947,
              'longitude': 5.4496794},
 'metadata': {'address': '9 Rue Pasteur, 38110 La Tour-du-Pin',
              'business_hours': {'dimanche': None,
                                 'jeudi': None,
                                 'lundi': None,
                                 'mardi': None,
                                 'mercredi': None,
                                 'samedi': None,
                                 'vendredi': None}},
 'nom': 'Centre de vaccination de la Tour du Pin',
 'plateforme': 'Doctolib',
 'prochain_rdv': '2021-06-02T11:10:00.000+02:00',
 'type': 'vaccination-center',
 'url': 'https://www.doctolib.fr/centre-de-vaccinations-internationales/la-tour-du-pin/centre-de-vaccination-de-la-tour-du-pin',
 'vaccine_type': ['Pfizer-BioNTech']}

```

**Comment by simonecid** [comment]
Hi,

I see. So I am creating false positives. I also noticed inconsistencies but did not look further on the reason why, thanks for digging out why and opening this issue. 
When I created this a few days ago, there was no "appointment_schedules" entry in the JSON. It must have been added with the new ChronoDose feature of ViteMaDose.
It looks fairly easy to dig out the correct info from the JSON. I will try to get this sorted by today.

Cheers,
Simone
